README.md

```diff
@@ -102,6 +104,10 @@ A curated list of awesome MATLAB toolboxes, applications, software and resources
 * [Wavelab](http://statweb.stanford.edu/~wavelab/Wavelab_850/index_wavelab850.html) - WaveLab is a collection of Matlab functions that have been used by the authors and collaborators to implement a variety of algorithms related to wavelet analysis.
 * [emgr](http://gramian.de) - Empirical gramian framework for model reduction and system identification.
 * [Matlab Toolbox for Dimensionality Reduction](https://lvdmaaten.github.io/drtoolbox/)
+* [MPITB](http://www.ugr.es/~jfernand/mpitb_eng.html) - MPI Toolbox for Matlab ([MPI Toolbox for Octave](http://www.ugr.es/~jfernand/mpitb.html)) 
+* [Lightspeed](http://research.microsoft.com/en-us/um/people/minka/software/lightspeed/) - Microsoft Lightspeed Utilities Toolbox
+* [Brain connectivity Toolbox](https://sites.google.com/site/bctnet/)
+* [OpenCL Toolbox](https://code.google.com/p/opencl-toolbox/)
```

Comment:
I have similar comments as for the blogs: it's not clear what those are and why these are here?  (e.g. what is MPI, LightSpeed, OpenCL)

Moreover, I have some doubts about:
 - MPITB. As I see, the website has not been changed since 2008/2009 and MPITB interacts with C code/Mex as far as I see. I would be astonished if that still works with recent versions of MATLAB. Do you still use this?

 - OpenCL Toolbox. This seems to be unmaintained for a while and Google Code is scheduled to reduce its access even further by the end of this month. I am exporting the repository to [GitHub](https://github.com/imaginairy-user/opencl-toolbox/) while still possible.

If you actively use the latter, I'm more than happy to transfer that repository to you and then it's probably best to link to that repository instead.